webhook/queue/worker.go

```diff
@@ -22,12 +24,11 @@ func ProcessWebhooks(ctx context.Context, webhookQueue chan redisClient.WebhookP
 				if err == nil {
 					break
 				}
-				log.Println("Error sending webhook:", err)
+				logging.WebhookLogger(logging.ErrorType, fmt.Errorf("error sending webhook: %s", err))
 
 				retries++
 				if retries >= maxRetries {
-					log.Println("Max retries reached. Giving up on webhook:", p.WebhookId)
-					break
+					logging.WebhookLogger(logging.WarningType, fmt.Errorf("max retries reached. Giving up on webhook: %s", p.WebhookId))
```

Comment:
@koladev32 you didn't stop the loop after the log or do you think the return function in the logging will stop it (am not sure though)

Reply:
corrected